iFixit/react-commerce
# Improve product list item pricing

fixes #259 

## Changelog

- If customer is a pro, use the normal `price` as the `compare_at_price` if `compare_at_price` is `null` in the Algolia index.

## QA

1. Visit the Vercel PR preview (replace `vercel.app` with `cominor.com`)
2. Visit the tools product list (`/Store/Tools`)
3. Verify that a pro user sees the normal price as a compare at price when a product has a discounted price for the user tier
